docs/UserGuide.md

```diff
@@ -82,17 +82,45 @@ In this comprehensive user guide, we will take you to experience a full journey
 ### 1.1 What is TrackMate
 <a id="what-is-trackmate"></a>
 
-TO BE DONE
+TrackMate is a desktop application specifically designed for teaching assistants (TAs) who work at the School of 
+Computing (SoC) of National University of Singapore (NUS). By providing an efficient solution to manage student data, 
+TrackMate simplifies the process of tracking assignments, tutorials, and attendance. With its powerful Command Line 
+Interface (CLI), TrackMate allows TAs to efficiently manage student information while enhancing productivity in their 
+administrative tasks.
+
+Overview of Main Features:
+* Tutorial Management: Track student tutorial groups for each session.
+* Attendance Monitoring: Record attendance effortlessly to ensure that student participation is logged accurately.
+* Assignment Tracking: Easily add, update, and manage assignment submissions and statuses for students.
```

Comment:
I like how you describe the main features, they do encapsulate all main features that we wish to deliver!